### PR TITLE
UHM-6229 - Download OCL package and remove MDS packages if configured…

### PR DESCRIPTION
**Pull request by mseaton**
… to do so.

### CODE REVIEW

File: mirebalais-modules/openmrs/manifests/pihemr.pp [hunk 1]
```diff
@@ -163,6 +164,29 @@
...
+          require => [ Exec['install-openmrs-configuration'], File["${tomcat_home_dir}/.OpenMRS"] ],
+          notify => [ Exec['tomcat-restart'] ]
+        }
+
+        wget::fetch { 'download-ocl-package-zip':
+          source      => "${ocl_package_url}",
+          destination => "{tomcat_home_dir}/.OpenMRS/configuration/ocl/",
+          timeout     => 0,
+          verbose     => false,
+          redownload  => true
```
Comment (mogoodrich) [inline mirebalais-modules/openmrs/manifests/pihemr.pp:181]: This likely needs the "File" require to confirm the .OpenMRS directory has been created?  (it's possible you may have to explicitly create "ocl" as well)

**Review by mogoodrich — APPROVED** [review]
One small comment, but LGTM @mseaton

File: mirebalais-modules/openmrs/manifests/pihemr.pp (hunk 1 above)
Comment (mseaton) [reply mirebalais-modules/openmrs/manifests/pihemr.pp]: OK, thanks, I pushed an extra commit to try to address this but may need to work through it a bit post-commit.

Comment (mogoodrich) [reply mirebalais-modules/openmrs/manifests/pihemr.pp]: Yeah, definitely easiest to test/tweak post-commit after you see how things run on OCL Test.

**Pull request merged**